src/brevitas/proxy/float_runtime_quant.py

```diff
@@ -109,3 +109,56 @@ def forward(self, x: Union[Tensor, QuantTensor]) -> Union[Tensor, FloatQuantTens
             cached_out = _CachedIOFloat(out.detach(), self.cache_quant_io_metadata_only)
             self._cached_act = cached_out
         return out
+
+
+class ActFloatQuantProxyFromInjector(ActFloatQuantProxyFromInjectorBase):
+
+    def forward(self, x: Union[Tensor, FloatQuantTensor]) -> Union[Tensor, FloatQuantTensor]:
+        out = x
+        if self.fused_activation_quant_proxy is not None:
+            y = x
+            if isinstance(y, FloatQuantTensor):
+                y = y.value
+
+            if self.export_mode:
+                y = self.fused_activation_quant_proxy.activation_impl(y)
+                y = self.export_handler(y)
+            elif not self.is_quant_enabled:
+                y = self.fused_activation_quant_proxy.activation_impl(y)
+            else:
+                y = self.fused_activation_quant_proxy(y)
+            # If y is an empty FloatQuantTensor, we need to check if this is a passthrough proxy,
+            # otherwise return a simple Tensor
+            # We exclude the last two values (inf_values and nan_values)
+            if isinstance(y, tuple) and not any(map(lambda f: f is None, y[:-2])):
+                out = FloatQuantTensor(*y, signed=self.is_signed, training=self.training)
+            elif self.is_passthrough_act:  # preserve scale/zp/bit/sign even without output quant
+                if isinstance(y, tuple):
+                    y = y[0]
+                if isinstance(x, FloatQuantTensor):
+                    out = FloatQuantTensor(
```

Comment:
I also think these are out-of-order. (and also at line 86 in this same file - can't comment there because you didn't change it, but I think worth fixing in this PR anyway)